### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by pritam-kr**
- added result page
- added category page
- added two questions of marvel quiz as a demo 
- added quiz's guide page

Screenshot (209)

Screenshot (210)

Screenshot (211)

Screenshot (212)

### CODE REVIEW

File: commonstyle.css [hunk 1]
```diff
@@ -12,11 +12,97 @@
...
+  cursor: pointer;
+  justify-content: flex-start;
+  padding: 1.5rem;
+  margin-bottom: 0.7rem;
+  background-color: var(--secondary-bg);
+  border-radius: 5rem;
+  transition: 0.3s all;
+}
+.option:hover {
+  background-color: #1c3353;
```
Comment (srishti-maurya) [inline commonstyle.css:95]: you may use color variables here

File: commonstyle.css [hunk 2]
```diff
@@ -12,11 +12,97 @@
...
+}
+
+.quit {
+  font-size: large;
+  color: var(--wrong-ans);
+}
+
+.right {
```
[the remark this answers is not in the document]
Comment (srishti-maurya) [reply commonstyle.css:103]: similarly to `wrong-clr` as well

File: commonstyle.css [hunk 3]
```diff
@@ -12,11 +12,97 @@
...
   --secondary-bg: #0b121b;
+  --correct-ans: #007953;
+  --wrong-ans: #af1615;
+}
+
+body {
+  background-color: var(--primary-color);
 }
 
 li {
```
Comment (srishti-maurya) [inline commonstyle.css:36]: you may avoid styling directly the tag `li` instead you can give a class to it

Comment (srishti-maurya) [reply commonstyle.css:36]: similarly to `body` tag as well

**Review by varnita619 — APPROVED** [review]
UI looks clean. Good to merge

File: style.css [hunk 4]
```diff
@@ -1,52 +1,47 @@
...
+}
+.images-card img {
+  border-radius: 1rem;
+}
+.card-content {
+  border-bottom-left-radius: 1rem;
+  border-bottom-right-radius: 1rem;
+  padding: 1.5rem;
+  text-align: center;
+  color: #fff;
```
Comment (mimansha-swarup) [inline style.css:32]: use variable

File: style.css [hunk 5]
```diff
@@ -1,52 +1,47 @@
...
+  flex-wrap: wrap;
+  gap: 10px;
+  justify-content: center;
+  align-items: center;
+}
+.images-card {
+  box-sizing: var(--box-shadow);
+  border-radius: 1rem;
+  background-color: var(--secondary-bg);
+  max-width: 400px;
```
Comment (mimansha-swarup) [inline style.css:22]: why not use rem or em?

File: css/result.css [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+.mb{
+    margin-bottom: 2rem;
+}
+
+.option {
+    cursor: pointer;
+    justify-content: flex-start;
+    padding: 1.5rem;
+    margin-bottom: 0.7rem;
+    /* background-color: var(--secondary-bg); */
```
Comment (mimansha-swarup) [inline css/result.css:23]: remove unnecessary comment

**Review by mimansha-swarup — COMMENTED** [review]
few suggestion

File: commonstyle.css [hunk 7]
```diff
@@ -12,11 +12,97 @@
...
+.large-heading {
+  width: fit-content;
+  margin: 2rem 0;
+  font-size: 3rem;
+  color: var(--text-color);
+}
+
+.material-icons-outlined {
+  font-size: 4rem;
+  font-weight: bold;
```
Comment (BelideNikhil) [inline commonstyle.css:61]: use variables for font size, font weight  if possible, you'll get that consistency

**Review by BelideNikhil — APPROVED** [review]

**Pull request merged**